alembic/versions/9478bbaf8010_add_reservedip_table.py

```diff
@@ -0,0 +1,39 @@
+"""add reservedip table
+
+Revision ID: 9478bbaf8010
+Revises: 6627b128bd5c
+Create Date: 2019-10-14 12:21:25.619884
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '9478bbaf8010'
+down_revision = '6627b128bd5c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('reservedip',
+    sa.Column('device_id', sa.Integer(), nullable=False),
+    sa.Column('ip', sqlalchemy_utils.types.ip_address.IPAddressType(length=50), nullable=True),
+    sa.Column('last_seen', sa.DateTime(), nullable=True),
+    sa.ForeignKeyConstraint(['device_id'], ['device.id'], ),
+    sa.PrimaryKeyConstraint('device_id')
+    )
+    op.create_index(op.f('ix_reservedip_device_id'), 'reservedip', ['device_id'], unique=False)
+    op.create_unique_constraint(None, 'joblock', ['jobid'])
```

Comment:
What is the purpose of this?

Reply:
For the reservedip table the relationship to the device is used as a primary key instead of adding a new id column, at least in the current use case of reserving management addresses this should be fine I think? Unless we need to reserve multiple management IPs for the same device at the same time for some reason
The second thing about a constraint on the joblock table must be something that I added after making the initial alembic revision for joblock and then missed because I didn't run alembic again until now

Reply:
Was only the joblock constraint I was interested in, the other looks good.